pkg/api/server/v1alpha2/records_test.go

```diff
@@ -557,3 +557,48 @@ func TestUpdateRecord(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteRecord(t *testing.T) {
+	srv, err := New(test.NewDB(t))
+	if err != nil {
+		t.Fatalf("failed to create temp file for db: %v", err)
```

Comment:
nit: i'd make this error message more opaque if possible. this might have failed for other reasons besides the temp file. we use this elsewhere:

```suggestion
		t.Fatalf("failed to create server: %v", err)
```